100.prefix-suffix.yaml

```diff
@@ -80,6 +80,8 @@
 
 - { noflag: no_suffix, ruleset: npackd,          namepat: "(.*?)(-?64)", setname: $1, addflavor: "64" }
 
+- { noflag: no_suffix, ruleset: noir,           namepat: "(.*)-git", ver: "git", setname: $1, rolling: true }
```

Comment:
There are no *-git packages in noir. There probably needs to be a 899 rule to mark `git` versions rolling in noir.

Reply:
Sorry about that, mistook 100 for 899